design-toolkit/components/scripts/token-migration.mjs

```diff
@@ -0,0 +1,197 @@
+  'fg-advisory-bold',
+  'fg-advisory-hover',
+  'fg-advisory-pressed',
+  'fg-normal-bold',
+  'fg-normal-hover',
+  'fg-normal-pressed',
+  'fg-serious-bold',
+  'fg-serious-hover',
+  'fg-serious-pressed',
+  'fg-critical-bold',
+  'fg-critical-hover',
+  'fg-critical-pressed',
+  'fg-a11y-on-accent',
+  'fg-a11y-on-utility',
+  'outline-static',
+  'outline-interactive',
+  'outline-interactive-hover',
+  'outline-interactive-pressed',
+  'outline-interactive-disabled',
+  'outline-accent-primary-bold',
+  'outline-accent-primary-hover',
+  'outline-accent-primary-pressed',
+  'outline-info-bold',
+  'outline-info-hover',
+  'outline-info-pressed',
+  'outline-advisory-bold',
+  'outline-advisory-hover',
+  'outline-advisory-pressed',
+  'outline-normal-bold',
+  'outline-normal-hover',
+  'outline-normal-pressed',
+  'outline-serious-bold',
+  'outline-serious-hover',
+  'outline-serious-pressed',
+  'outline-critical-bold',
+  'outline-critical-hover',
+  'outline-critical-pressed',
+
+  // built ins that can be ignored
+  'bg-transparent',
+  'outline-none',
+  'outline-transparent',
+  'bg-initial',
+]);
+
+// from https://www.figma.com/design/9kqcWHdToDaWhvZSpkCO7c/CORE-DS-Design-Initiatives?node-id=56130-400&t=tqBFJhWrvp5yIWt9-4
+const replacementMap = new Map([
+  // bgs
+  ['bg-transparent-dark', 'bg-surface-muted'],
+  ['bg-interactive-default', 'bg-interactive-muted'],
+  ['bg-interactive-hover-light', 'bg-interactive-muted-hover'],
+  ['bg-highlight-subtle', 'bg-accent-primary-muted'],
+  ['bg-info-subtle', 'bg-info-muted'],
+  ['bg-advisory-subtle', 'bg-advisory-muted'],
+  ['bg-normal-subtle', 'bg-normal-muted'],
+  ['bg-serious-subtle', 'bg-serious-muted'],
+  ['bg-critical-subtle', 'bg-critical-muted'],
+  // fgs
+  ['fg-default-light', 'fg-primary-bold'],
+  ['fg-default-dark', 'fg-primary-muted'],
+  ['fg-inverse-light', 'fg-inverse-bold'],
+  ['fg-inverse-dark', 'fg-inverse-muted'],
+  ['fg-highlight', 'fg-accent-primary-bold'],
+  ['fg-info', 'fg-info-bold'],
+  ['fg-advisory', 'fg-advisory-bold'],
+  ['fg-normal', 'fg-normal-bold'],
+  ['fg-serious', 'fg-serious-bold'],
+  ['fg-critical', 'fg-critical-bold'],
+  // outline
+  ['outline-static-light', 'outline-static'],
+  ['outline-highlight', 'outline-accent-primary-bold'],
+  ['outline-info', 'outline-info-bold'],
+  ['outline-advisory', 'outline-advisory-bold'],
+  ['outline-normal', 'outline-normal-bold'],
+  ['outline-serious', 'outline-serious-bold'],
+  ['outline-critical', 'outline-critical-bold'],
+  // border
+  ['border-static-light', 'outline-static'],
+  ['border-highlight', 'outline-accent-primary-bold'],
+  ['border-info', 'outline-info-bold'],
+  ['border-advisory', 'outline-advisory-bold'],
+  ['border-normal', 'outline-normal-bold'],
+  ['border-serious', 'outline-serious-bold'],
+  ['border-critical', 'outline-critical-bold'],
+]);
+
+const leGlob = 'src/**/*.{ts,tsx}';
+console.log(`🔍 Finding and replacing in ${leGlob}`);
+console.log();
```

Comment:
lots of console logs in here that seem helpful for a script output except for this empty lil guy